src/helpers/helper.js

```diff
@@ -1,6 +1,44 @@
+import { TOP } from '../constants/constants';
+
 // id generator
 // returns a string
 export const generateUniqueID = () => {
   const s4 = () => Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1);
   return `${s4()}-${s4()}-${s4()}-${s4()}`;
 };
+
+export const getFavouritesList = (array) => {
```

Comment:
You can rename array to contactsList. This function seems to be specific to contacts rather than for general arrays